migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql

```diff
@@ -106,16 +107,16 @@ update_statement := ' update
 
 --- update Condition
 update_condition := 'target.healthinsurancepayorid != source.healthinsurancepayorid
-        or target.healthinsuranceplantoplantypeid != source.healthinsuranceplantoplantypeid
-        or target.payorcode != source.payorcode
-        or target.payorname != source.payorname
-        or target.plandisplayname != source.plandisplayname
-        or target.planname != source.planname
-        or target.plantypedescription != source.plantypedescription
-        or target.plantypedisplaydescription != source.plantypedisplaydescription
-        or target.productname != source.productname
-        or target.providerid != source.providerid
-        or target.searchable != source.searchable';
+                            or target.healthinsuranceplantoplantypeid != source.healthinsuranceplantoplantypeid
+                            or target.payorcode != source.payorcode
+                            or target.payorname != source.payorname
+                            or target.plandisplayname != source.plandisplayname
+                            or target.planname != source.planname
+                            or target.plantypedescription != source.plantypedescription
+                            or target.plantypedisplaydescription != source.plantypedisplaydescription
+                            or target.productname != source.productname
+                            or target.providerid != source.providerid
+                            or target.searchable != source.searchable';
 
 --- insert Statement
 insert_statement := ' insert
```

Comment:
the HealthInsurancePayorID should be created as a uuid_string here in the insert not in the select

Reply:
This is a Mid table so we do not insert ids here.